cmd/lakectl/cmd/repo_create.go

```
@@ -27,13 +27,19 @@ var repoCreateCmd = &cobra.Command{
 		clt := getClient()
 		u := MustParseRepoURI("repository URI", args[0])
 		fmt.Println("Repository:", u)
-		defaultBranch := Must(cmd.Flags().GetString("default-branch"))
-		sampleData := Must(cmd.Flags().GetBool(SampleDataFlag))
+
+		defaultBranch, err := cmd.Flags().GetString(defaultBranchFlagName)
+		if err != nil {
+			DieErr(err)
+		}
+		sampleData := Must(cmd.Flags().GetBool(sampleDataFlagName))
+		storageID, _ := cmd.Flags().GetString(storageIDFlagName)
```

Why ignore the error?

---

There are other possible errors than "flag accessed but not defined"

---

Thanks.
Added error handling for this (obviously, copied from other commands).